psalm.xml

```diff
@@ -5,14 +5,35 @@
     xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
     name="Example Psalm config with recommended defaults"
     useDocblockTypes="true"
-    totallyTyped="false"
+    totallyTyped="true"
     allowPhpStormGenerics="true"
 >
     <projectFiles>
         <directory name="src"/>
     </projectFiles>
 
     <issueHandlers>
-        <PropertyNotSetInConstructor errorLevel="suppress"/>
+        <PropertyNotSetInConstructor>
```

Comment:
Named constructors will be used in every value type in the library (and tbh in all the new code I'm writing these days): does this check make any sense at all? Are named constructors something that the library could work with? Effectively, the rule should be something like "complete this object before it is passed to anything else", rather than "complete this object in the constructor".

Reply:
Yeah, I'll get rid of this if it's not how you write code.

> Are named constructors something that the library could work with?
> Effectively, the rule should be something like "complete this object before it is passed to anything else", rather than "complete this object in the constructor".

I think it could be fairly easily supported as long as it only applied to classes with

```
private function __construct {}
```

There'd be some other error (`PropertyNotSetAfterInitialization` or similar) in that circumstance.

Reply:
Yeh that would make sense :+1: